### PR TITLE
Document missing commands and permissions. 

### LINKED ISSUE #618: SpongeCommon/SpongeForge command changes
Just wanted to note a few changes to the `/sponge plugins` command.

As of now SpongeCommon uses pagination for the plugin command to display installed plugins. On SpongeCommon/SpongeVanilla it forces `Minecraft, MCP, SpongeAPI, and Sponge` at the beginning of the list in that order. The console will display a non-paginated list for this command.

SpongeForge filters out mods from the `/sponge plugins` command and added a new command `/sponge mods` which will display all mods. This forces `Minecraft, MCP, FML, MinecraftForge, SpongeAPI, and Sponge` at the beginning of the list in that order. The console also displays a non-paginated list for this command. This command uses the permission `sponge.command.mods`.

Both commands also make use of tooltips for each listed item that allow you to hover over the plugin/mod and see the ID and Version quickly. Clicking it still acts the same by showing a more detailed output of the plugin/mod's properties.

**tl;dr**: `/sponge plugins` now is a paginated list when sent to players. Filters out mods on SpongeForge. SpongeForge added `/sponge mods` which displays installed mods and uses the permission `sponge.command.mods`

### LINKED ISSUE #722: commands page missing selector permission node
apparently its a thing added years ago 

https://github.com/SpongePowered/SpongeCommon/blob/a9423c7ec9f2b423b52c273f7e7a23818a8f3e41/src/main/java/org/spongepowered/common/command/CommandPermissions.java#L46

https://github.com/SpongePowered/SpongeCommon/pull/940

but its not mentioned on the commands page https://docs.spongepowered.org/stable/en/server/spongineer/commands.html

### PR DESCRIPTION
**Pull request by ST-DDT**
Fixes #618
Fixes #722

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/ba271c6/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/ba271c6/server/spongineer/commands.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/747#issuecomment-415500579), not to the direct url to the preview._

File: source/server/spongineer/commands.rst [hunk 1]
```diff
@@ -26,30 +26,43 @@ Sponge
...
-/sponge config          Alters a global, world, or a dimension    sponge.command.config
+/sponge config          Alters a global, world, or a dimension     sponge.command.config
                         config.
-/sponge heap            Dumps the JVM heap.                       sponge.command.heap
-/sponge help            View information on commands used on the  sponge.command.help
-                        server.
-/sponge plugins         Lists currently installed plugins.        sponge.command.plugins
-/sponge plugins reload  Asks plugins to perform their own reload  sponge.command.plugins.reload
+/sponge entityinfo      Shows the type and the some additional     sponge.command.entityinfo
+                        information about the block you are
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: block -> entity

File: source/server/spongineer/commands.rst [hunk 2]
```diff
@@ -26,30 +26,43 @@ Sponge
...
+/sponge timings         The main command for the timings module.   sponge.command.timings
+/sponge tps             Display ticks per second for each world.   sponge.command.tps
+/sponge version         Prints the Sponge/SpongeAPI versions to    sponge.command.version
                         the console.
-======================  ========================================  =============================
+/sponge which           Prints which plugin provided the command,  sponge.command.which
+                        it's aliases and alternatives.
+/sponge:callback        Internally used for callback actions on 
+                        ``Text``\s (such as pagination). Not
+                        meant to be invoked by hand.
```
Comment (Inscrutable) [inline source/server/spongineer/commands.rst]: meant -> intended

**Review by Inscrutable — CHANGES_REQUESTED** [review]
A whoops and a quibble. Neat-O.

File: source/server/spongineer/commands.rst [hunk 3]
```diff
@@ -1,6 +1,6 @@
-========
-Commands
-========
+===============
+Ingame Commands
```
Comment (phit) [inline source/server/spongineer/commands.rst]: why? commands can be used from console too and not just in game, honestly if anything this page should be called "Commands and Permissions"

Comment (ST-DDT) [reply source/server/spongineer/commands.rst]: I first though about calling it `Server Commands` to separate it from `Plugin Commands`.

**Review by phit — APPROVED** [review]
lgtm, the permissions in the title is nice because people can never find the permission nodes and dont think to check on the commands page

**Review by Inscrutable — APPROVED** [review]

**Pull request merged**